packages/databyss-services/sources/SourceProvider.js

```diff
@@ -44,9 +53,7 @@ const SourceProvider = ({ children, initialState, reducer }) => {
     }
   }
 
-  const getSources = () => {
-    return state.cache
-  }
+  const getSources = () => state.cache
```

Comment:
This method is confusing. Do we need it?

Reply:
Only used in `EditorPage`, but we want to use `getPageSources` instead there. 
Also used in `fetchAllSources`, which we can remove

Reply:
This method is needed to check the state of the cache for composing a list of sources not already in the cache. Ill change this to `getCache`

Reply:
i changed this from `getSources()` to `state`